### ISSUE TITLE
Dependencies in both dev and main dependencies

### ISSUE THREAD

**Issue by myndzi** [konobi/kwalitee#16]
NPM warns about this, but it can be reasonably considered a quality-worthy consideration. Also, this module fails it.

**Comment by myndzi** [comment]
(and, apparently, its own test suite...?)

**Comment by konobi** [comment]
yup! I have this already in mind... the "addon" checker has some dependencies vs devDependencies checks to ensure proper declarations.
